src/KOKKOS/fix_nh_kokkos.cpp

```diff
@@ -289,7 +295,7 @@ void FixNHKokkos<DeviceType>::remap()
   double oldlo,oldhi;
   double expfac;
 
-  int nlocal = atom->nlocal;
+  int nlocal = atomKK->nlocal;
```

Comment:
Not necessary, the variable is the same.

Reply:
done